src/reducer/forgot password/forgotPassword.test.js

```diff
@@ -0,0 +1,22 @@
+import forgotPasswordReducer from './forgotPassword.reducer';
```

Comment:
I believe you can optimize the use of variable names while you still ensure that the names remain clear but not necessarily long.

Reply:
Thanks for the feedback. I believe the naming still looks good